site/content/gitops-quickstart/01-apply-gitops.md

```diff
@@ -0,0 +1,350 @@
+---
+title: Setup your cluster with GitOps
+weight: 10
+url: gitops-quickstart/setup-gitops
+---
+
+# Setup your cluster with GitOps
+
+Welcome to `eksctl` GitOps Quick Starts. In this guide we will show you
+how to launch fully-configured Kubernetes clusters that are ready to
+run production workloads in minutes: easy for you to get started running
+Kubernetes on EKS and to launch standard clusters in your organisation.
+
+At the end of this, you will have a Kubernetes cluster including control
+plane, worker nodes, and all of the software needed for code deployment,
+monitoring, and logging.
+
+## Quick Start to GitOps
+
+[GitOps][gitops] is a way to do Kubernetes application delivery. It
+works by using Git as a single source of truth for Kubernetes resources
+and everything else. With Git at the center of your delivery pipelines,
+you and your team can make pull requests to accelerate and simplify
+application deployments and operations tasks to Kubernetes.
+
+[gitops]: https://www.weave.works/technologies/gitops/
+
+Using GitOps Quick Starts will get you set up in next to no time. You
+will benefit from a setup that is based on the experience of companies
+who run workloads at scale.
+
+## Prerequisites
+
+To use EKS, you need to have your [AWS account][aws-account] set up.
+
+Next you will have to have the following tools installed:
+
+- [AWS CLI][aws-cli]: at least `1.16.156` - older versions will require
```

Comment:
It's not required at all, please remove from here.

Reply:
https://github.com/weaveworks/eksctl/issues/1109

@cPu1 made me aware of this.

Reply:
Yes, but it's only need if you don't have `aws-iam-authenticator` and below you are asking for that one too. It's easier to install `aws-iam-authenticator` as it's a single binary, AWS CLI is a Python package and can be tricky if you don't have a clean Python installation.